src/hostapi/wasapi/pa_win_wasapi.c

```diff
@@ -6087,10 +6096,11 @@ static UINT32 ConfigureLoopSleepTimeAndScheduler(PaWasapiStream *stream, ThreadI
 }
 
 // ------------------------------------------------------------------------------------------
-static inline INT32 GetNextSleepTime(SystemTimer *timer, ThreadIdleScheduler *scheduler, LONGLONG startTime,
+static PA_INLINE INT32 GetNextSleepTime(SystemTimer *timer, ThreadIdleScheduler *scheduler, LONGLONG startTime,
     UINT32 sleepTime)
 {
     INT32 nextSleepTime;
+	UINT32 procTime;
```

Comment:
Same here

Reply:
`UINT32 procTime` must be changed back to `INT32 procTime`.

Reply:
it was a typo, i will change it back to INT32